README.md

```diff
@@ -25,7 +25,12 @@ Play!
 ## How to build it (if you want to change the code)
 The simplest, though most time consuming, option is to run:
 
-	make
+```
```

Comment:
I like this section but it's not quite what I meant (which means it definitely needs some overhauling). I meant it to be how to build it each time, whereas you've added first time instructions. Could you maybe separate them into two sections?